client/webpack.config.js

```diff
@@ -1,39 +1,36 @@
 // webpack.config.js
-const webpack = require("webpack")
-const path = require("path")
-const UglifyJSPlugin = require("uglifyjs-webpack-plugin")
-const ExtractTextPlugin = require("extract-text-webpack-plugin")
-const CopyWebpackPlugin = require("copy-webpack-plugin")
+const webpack = require("webpack");
```

Comment:
...I just hit npm install and prettier wants to do a lot. Is this expected @magsout?

Reply:
hum, no sure to get it. What do you mean by `wants to do a lot` ?